src/librustc/mir/interpret/error.rs

```diff
@@ -374,18 +396,11 @@ pub enum InterpError<'tcx> {
     HeapAllocZeroBytes,
     HeapAllocNonPowerOfTwoAlignment(u64),
     Unreachable,
+    /// The program panicked.
     Panic(PanicMessage<u64>),
```

Comment:
Please move this up to the other "here to stay" variants -- and then have a clear separator.

Then we can actually move variants to the inner enums piecewise, which seems nice if we don't want this PR to become too big!